packages/Heading/Heading.jsx

```diff
@@ -30,17 +30,16 @@ BaseHeading.propTypes = {
  *
  * @version ./package.json
  */
-const Heading = ({ level, invert, children, ...rest }) => {
+const Heading = ({ level, size = level, invert, children, ...rest }) => {
```

Comment:
We tend to not use default assignment when defining React components in favor of using defaultProps so that the default value gets automatically documented by react-styleguidist.

In this case I see why that's not possible though since the default value is dynamic.